squirrel.frontier/src/test/java/org/dice_research/squirrel/frontier/impl/FrontierImplTest.java

```diff
@@ -27,38 +31,37 @@
 public class FrontierImplTest {
 
 
+
     private static FrontierImpl frontier;
     private static MongoDBIpBasedQueue queue;
     private static MongoDBKnowUriFilter filter;
     private static List<CrawleableUri> uris = new ArrayList<CrawleableUri>();
     private static CrawleableUriFactory4Tests cuf = new CrawleableUriFactory4Tests();
+    private OutDatedUriRetreiver outDatedUriRetreiver = SparqlhostConnector.create("http://localhost:8890/sparql-auth", "dba", "pw123");
```

Comment:
This line increases the complexity of the test and it is not necessary. Please use an in-memory SPARQL endpoint, e.g., based on a `Dataset` instance provided by the Apache Jena library which can be used to mimik the SPARQL endpoint.